entities/src/api_req_params.rs

```diff
@@ -264,26 +226,22 @@ pub struct GetAssetSignatures {
     pub leaf_index: Option<u64>,
     #[serde(default)]
     pub sort_direction: Option<AssetSortDirection>,
+    #[serde(default)]
     pub cursor: Option<String>,
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
-#[serde(rename_all = "camelCase")]
-pub struct DisplayOptions {
-    pub show_zero_balance: bool,
-}
-
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema, Default)]
 #[serde(deny_unknown_fields, rename_all = "camelCase")]
 pub struct GetTokenAccounts {
+    pub owner_address: Option<String>,
```

Comment:
Why do we change `owner` and `mint` parameters names?

Reply:
to align with the reference DAS 🫡  https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/blob/8db88a829474c230eb9a3ff33e58ecead9c08fb8/das_api/src/api/mod.rs#L183